pkgs/racket-doc/scribblings/reference/hashes.scrbl

```diff
@@ -214,16 +214,6 @@ mappings can hide earlier mappings.
 
 See also @racket[make-custom-hash].
 
-@examples[
-#:eval the-eval
-(make-hash)
-(make-hash '([0 . 1] [42 . "meaning of life"] [2 . 3]))
-(make-hash '([0 . 1] [1 . 2] [0 . 3]))
-(make-hash (list (cons 0 1) (cons 'apple 'orange) (cons #t #f)))
-(make-hash '((0 1) (1 2) (2 3)))
-(make-hash (list (cons + -)))
-]
-
```

Comment:
Why delete these examples for `make-hash`, and also `hash-map` below?

Reply:
I f-ed it up after rebasing to resolve conflicts. Sorry. Redoing it.